js/main.js

```diff
@@ -172,8 +176,49 @@ $ul.addEventListener('click', function (event) {
       $form.elements.notes.value = data.editing.comment;
       $imagePlaceholder.setAttribute('src', data.editing.url);
       $entryFormText.textContent = 'Edit Entry';
-
       viewSwap('entry-form');
+
+      $deleteButton.className = 'delete-button';
     }
   }
 });
+
+// modal below
+const $deleteBtn = document.querySelector('.delete-button');
+const $popupModal = document.querySelector('.delete-entry-background');
+$deleteBtn.addEventListener('click', $deleteEntryPopup);
+function $deleteEntryPopup() {
+  $popupModal.className = 'delete-entry-background';
+}
+
+// When the user clicks Cancel, hide the modal.
+const $cancelBtn = document.querySelector('.dlt-btn-cancel');
+$cancelBtn.addEventListener('click', $cancelDelete);
+function $cancelDelete() {
```

Comment:
`$` should be used alongside variables that hold DOM elements. They shouldn't be used for function names